e2e/sample.e2e-spec.ts

```diff
@@ -1,20 +0,0 @@
-import {
```

Comment:
Could you also delete the `./src/app/public/styles` folder?

Reply:
done